.eslintrc.json

```diff
@@ -1,7 +1,4 @@
 {
-    "extends": ["@idearium/eslint-config", "plugin:prettier/recommended"],
-    "parserOptions": {
-        "ecmaVersion": 2017
-    },
-    "root": true
+    "root": true,
```

Comment:
@allanchau, don't we need `prettier/recommended` to remove styling rules from our config?

Reply:
@smebberson I've fixed that up now.

Reply:
@allanchau, also, why did you remove the `ecmaVersion`? I'm sure that was only added if required? I think it's to allow `async () => {}`. You can check it being incorrectly reported in `bin/lib/c.js` at L339.

Reply:
@smebberson I don't remember why I changed this file, I probably copied it from another project. I've reverted the change now.

Reply:
Cool.